tests/plugins/test_change_from_in_df.py

```diff
@@ -69,42 +70,15 @@ def test_update_base_image(tmpdir, workflow, base_image):
                              df_path=dfp.dockerfile_path,
                              df_images=dfp.parent_images)
     workflow.dockerfile_images[base_image] = local_tag
-    (flexmock(workflow.imageutil)
-     .should_receive('get_inspect_for_image')
-     .with_args(local_tag)
-     .and_return(dict(Id=base_str)))
 
     run_plugin(workflow)
     expected_df = df_content.format(base_str)
     assert dfp.content == expected_df
 
 
-def test_update_base_image_inspect_broken(tmpdir, workflow, caplog):
-    """exercise code branch where the base image inspect comes back without an Id"""
-    df_content = "FROM base:image"
-    dfp = df_parser(str(tmpdir))
-    dfp.content = df_content
-    image_str = "base@sha256:1234"
-    local_tag = ImageName.parse(image_str)
-    workflow = mock_workflow(workflow,
-                             df_path=dfp.dockerfile_path,
-                             df_images=dfp.parent_images)
-    workflow.dockerfile_images['base:image'] = local_tag
-    (flexmock(workflow.imageutil)
-     .should_receive('get_inspect_for_image')
-     .with_args(local_tag)
-     .and_return(dict(no_id="here")))
-
-    with pytest.raises(PluginFailedException) as exc:
-        run_plugin(workflow)
-    assert "raised an exception: NoIdInspection" in str(exc.value)
-    assert dfp.content == df_content  # nothing changed
-    assert "missing in inspection" in caplog.text
-
-
-@pytest.mark.parametrize(('df_content, expected_df_content'), [
+@pytest.mark.parametrize('df_content, expected_df_content', [
     (
-        dedent("""\
+            dedent("""\
             FROM first:parent AS builder1
```

Comment:
can we have also some test with full  pullspec?  registry/image/tag

Reply:
added test with full pullspec